src/dom/Value.ts

```diff
@@ -289,6 +295,10 @@ export function Value<Clazz extends Constructor>(BaseClass: Clazz, ionType: IonT
             this._unsupportedOperation('writeTo');
         }
 
+        deleteField(name: string): boolean {
+            this._unsupportedOperation('delete');
```

Comment:
The rename from `delete` to `deleteField` missed this string:
```suggestion
            this._unsupportedOperation('deleteField');
```